src/azure-cli-core/setup.py

```diff
@@ -50,8 +50,8 @@
     'humanfriendly>=4.7,<10.0',
     'jmespath',
     'knack==0.8.0rc2',
-    'msal~=1.0.0',
-    'msal-extensions~=0.1.3',
+    'msal~=1.3',
+    'msal-extensions~=0.2',
```

Comment:
Do we really need `msal-extensions`? I don't see it being used anywhere.

Reply:
Removed and verified.